From f6163532c6a85658b51960377659bfca6c54fc63 Mon Sep 17 00:00:00 2001
From: Shubhrajyoti Datta <shubhrajyoti.datta@xilinx.com>
Date: Fri, 14 Feb 2020 12:55:13 +0530
Subject: [PATCH 1197/1852] dt-bindings: usb: Add clock support for udc-xilinx

commit e6c6e966dee5b8853367c50715b1da3e9d031483 from
https://github.com/Xilinx/linux-xlnx.git

Add clock support for xilinx udc driver. While at it fix the case
for xlnx,has-builtin-dma.

Signed-off-by: Shubhrajyoti Datta <shubhrajyoti.datta@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 Documentation/devicetree/bindings/usb/udc-xilinx.txt | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/Documentation/devicetree/bindings/usb/udc-xilinx.txt b/Documentation/devicetree/bindings/usb/udc-xilinx.txt
index 47b4e397a08d..33847fef467e 100644
--- a/Documentation/devicetree/bindings/usb/udc-xilinx.txt
+++ b/Documentation/devicetree/bindings/usb/udc-xilinx.txt
@@ -6,7 +6,11 @@ Required properties:
 			  device registers map.
 - interrupts		: Should contain single irq line of USB2 device
 			  controller
-- xlnx,has-builtin-dma	: if DMA is included
+- xlnx,has-builtin-dma	: If DMA is included
+
+Optional properties:
+- clock-names		: Should be  "s_axi_aclk"
+- clocks		: Input clock specifier. Refer to common clock bindings.
 
 Example:
  		axi-usb2-device@42e00000 {
-- 
2.31.1

